examples/dreamcast/libdream/keyboard/keyboard.c

```diff
@@ -28,16 +30,8 @@ void kb_test(void) {
             return;
         }
 
-        thd_sleep(10);
-
-        /* Check for keyboard input */
-        /* if (kbd_poll(mkb)) {
-            printf("Error checking keyboard status\n");
-            return;
-        } */
-
         /* Get queued keys */
-        while((k = kbd_queue_pop(kbd, 1)) != -1) {
+        while((k = kbd_queue_pop(kbd, 1)) != KBD_QUEUE_END) {
             if(k == 27) {
```

Comment:
Values like '27' should really be inside enum.
```
enum class EKeyboardKeys 
{
      KB_KEY_ESCAPE = 27
}
```

Reply:
There's an enum for all of the keyboard keys, but this API is special in that it's not popping off the actual keyboard key number, it's popping off the ASCII value the key corresponds to, hence 27... but yeah, probably at very least this could be a `#define` or an enum or even a comment... fair.